build.sh

```diff
@@ -29,12 +29,20 @@ fi
 
 if [ $(DOCKER_CLI_EXPERIMENTAL=enabled docker manifest inspect $build_image_name 2> /dev/null ; echo $?) -eq 0 ]; then
 	echo "----> Skipping build as image already exists"
-else
+	exit 0
+fi
+
+if [ -f Dockerfile ]; then
+        docker build -t ${build_image_name%:*}:latest -t $build_image_name .
+	build_built=true
+fi
+	
+if [ -f project.toml ]; then
```

Comment:
This will run together with the 35 line, if any project has Dockerfile (to local development) and Project.toml to build.

Reply:
@fagianijunior having them both is definitely not ideal but how to handle that? which one takes precedence? I dislike the idea of adding yet another ENV for defining it as proposed in the last commit.